typescript/best-practice-web/test/pages/login.page.ts

```diff
@@ -0,0 +1,30 @@
+import Page from './page';
+
+class LoginPage extends Page {
+    /**
+     * The current implementation uses the locators inside of the method.
+     * As the page object expands, you will find that you are reusing the 
+     * locators in multiple places. In that case, you might want to move
+     * them into a private property
+     */
+    login (userDetails):void {
```

Comment:
For Typescript `userDetails` needs to be typed, it's not here, so can you add that, otherwise it would by default get the `any` type and that is not what we want with a best practice 😉